test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -915,7 +914,7 @@ void ProtoConverter::createFunctionDefAndCall(T const& _x, unsigned _numInParams
 	// Manually create a multi assignment using global variables
 	// This prints a_0, ..., a_k-1 for this function that returns "k" values
 	if (_numOutParams > 0)
-		m_output << YulUtilFunctions::suffixedVariableNameList("a_", 0, _numOutParams) << " := ";
+                m_output << dev::suffixedVariableNameList("a_", 0, _numOutParams) << " := ";
```

Comment:
still spaces :(

Reply:
argh. this time I am sure that my editor is configured correctly (and I checked the diff not only locally, but also the one here (which I should have been doing before too, sry))